inst/json/sample_scenario_results_response.json

```diff
@@ -0,0 +1,86 @@
+{
+  "runId": "007e5f5453d64850",
+  "costs": [{
+    "id":  "total",
+    "value": 27000000,
+    "children": [
+      {
+        "id": "gdp",
+        "value": 4000000,
+        "children": [
+          {
+            "id": "gdp_closures",
+            "value": 1000000
+          },
+          {
+            "id": "gdp_absences",
+            "value": 3000000
+          }
+        ]
+      },
+      {
+        "id": "education",
+        "value": 6000000,
+        "children": [
+          {
+            "id": "education_closures",
+            "value": 2000000
+          },
+          {
+            "id": "education_absences",
+            "value": 4000000
+          }
+        ]
+      },
+      {
+        "id": "life_years",
+        "value": 17000000,
+        "children": [
+          {
+            "id": "life_years_infants",
+            "value": 1000000
+          },
+          {
+            "id": "life_years_adolescents",
+            "value": 2000000
+          },
+          {
+            "id": "life_years_working_age",
+            "value": 5000000
+          },
+          {
+            "id": "life_years_retirement_age",
+            "value": 9000000
+          }
+        ]
+      }
+    ]
+  }],
+  "capacities": [
+    {
+      "id": "hospital_capacity",
+      "value": 40000
+    },
+    {
+      "id": "icu_capacity",
+      "value": 5000
+    }
+  ],
+  "interventions": [
+    {
+      "id": "school_closures",
+      "start": 1,
+      "end": 4
+    },
+    {
+      "id": "business_closures",
+      "start": 3,
+      "end": 8
+    }
+  ],
```

Comment:
As you say, I think the strategy itself is implicit from the requested parameters, but it sounds like we should include implementation level (light/heavy) of that strategy in these intervention sections. 

Maybe we should actually include the requested parameters in the response too. It's superfluous but might help with debugging/sanity checks.

Reply:
I guess it depends a little bit on whether the requested parameters will be available for use alongside the output. As a silly example: request a model run, and display the parameters alongside/overlaid on a plot of the output.

If the parameters are already available for use in this way, then returning top-level values (e.g. country, infection, response strategy) as sanity checks should be enough.

Reply:
The web app should really be keeping track of what parameters go with what runIds and result sets, but I don't think it would hurt to include them in the result too.